src/components/PhotoList.jsx

```diff
@@ -57,4 +57,24 @@ export class PhotoList extends Component {
   }
 }
 
-export default Dimensions()(PhotoList)
+export class PhotoLists extends Component {
+  render () {
+    const { photoLists, containerWidth, selected, onPhotoToggle } = this.props
+    return (
+      <div>
+        { photoLists.map(photoList => {
+          return (<PhotoList
```

Comment:
@gregorylegarec In that way we must not create a *just add a `div`* component, but refactor the `render` function of the parent component to integrate this div with something like that, for instance:

```js
render () {
  const lists = photoLists.map((photoList) => (<PhotoList key={photoList.title} />))
  const withDimensions = Dimensions()(<div>lists</div>)
  return (
    <div className='photoBoard'>
      {!isBusy && withDimensions}
    </div>
  )
}